app/main.py

```diff
@@ -2,0 +1,17 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
```

Comment:
This is a good example of using parentheses to import multiple classes. It makes the code cleaner and easier to read.